src/shared/components/post/post-form.tsx

```diff
@@ -553,7 +565,15 @@ export class PostForm extends Component<PostFormProps, PostFormState> {
   }
 
   handlePostUrlChange(i: PostForm, event: any) {
-    i.setState(s => ((s.form.url = event.target.value), s));
+    const url = event.target.value;
+
+    i.setState({
+      form: {
+        url,
+      },
+      imageDeleteUrl: "",
```

Comment:
If `form.url` is changed, we can assume user wants to abandon image – so `imageDeleteUrl` should be cleared.

Reply:
Would be really nice to add an uploaded media manager someday where all uploads are tracked in `LocalStorage`. 🤔